archive/a-data-driven-thanksgiving.md

```diff
@@ -7,7 +7,7 @@ authors:
   - Dave Paprocki
 date: 2016-11-23T00:00:00.000Z
 ---
-
+<!-- markdownlint-disable-file -->
```

Comment:
Are these being disabled because they are "archives"? If so, seems like utilizing a Yarn script scoped to the `./posts/` directory would miss this file all together (no need to add inline ignores)?

Reply:
Yeah the scripts I was using already ignore everything outside of /posts. I just added these while I was figuring it out.